src/templates/partials/product.hbs

```diff
@@ -4,7 +4,7 @@
     </a>
     <h2>{{product.name}}</h2>
     <p class="price">${{product.price}}</p>
-    <form action="/{{product.url}}/cart" method="post">
+    <form action="/add/{{product.url}}" method="post">
```

Comment:
Please don't change the URL structure in a Pr related to styling. This is a huge change - e.g. influences all caching strategy. It is also inconsistent with /:id for product url and REST principles.

Reply:
I take your point re muddying PRs.

As for the URL, how else to implement remove-from-basket?

Reply:
E.g.

/{{product.url}}/cartAdd   |  /{{product.url}}/cartRemove or
/cart/add?id={{product.url}} | /cart/remove?id={{product.url}}

Depending where the logic is (in Product or Cart route)